Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -76,7 +80,8 @@ def __init__(self, indicator_type: str, api_token: str, services: list, risk_rul
         self.api_token = self.headers['X-RFToken'] = api_token
         self.services = services
         self.indicator_type = indicator_type
-        self.threshold = int(threshold) if threshold else threshold
+        self.malicious_threshold = int(malicious_threshold) if malicious_threshold else malicious_threshold
+        self.suspicious_threshold = int(suspicious_threshold) if suspicious_threshold else suspicious_threshold
```

Comment:
Let's add a test here to ensure that malicious_threshold > suspicious_threshold and if not throw an error.

Reply:
Completed!